automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingApiSpec.scala

```diff
@@ -0,0 +1,130 @@
+package org.broadinstitute.dsde.test.api
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import org.broadinstitute.dsde.workbench.auth.{AuthToken, AuthTokenScopes}
+import org.broadinstitute.dsde.workbench.config.{Credentials, ServiceTestConfig, UserPool}
+import org.broadinstitute.dsde.workbench.fixture._
+import org.broadinstitute.dsde.workbench.model.{UserInfo, WorkbenchEmail, WorkbenchUserId}
+import org.broadinstitute.dsde.workbench.service.BillingProject.BillingProjectRole
+import org.broadinstitute.dsde.workbench.service.{Google, Orchestration, Rawls, RestException}
+import org.scalatest.concurrent.Eventually
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+
+class BillingApiSpec extends FreeSpec with BillingFixtures with MethodFixtures with SubWorkflowFixtures
+  with Matchers with Eventually {
+
+  /**
+    * This test does
+    *
+    * Given) a registered user who is project owner
+    * and)   the project owner can access to the Google billing account
+    * When)  the project owner is authenticated with access token
+    * Then)  the project owner can create a new Google billing project
+    * and)   the project owner can add another user to Google billing project
+    * and)   the project owner can create a new workspace with Google billing project
+    * and)   the project owner can create new method and method config
+    * and)   the project owner can run an analysis and wait for complete successfully
+    * and)   the project owner can delete method config
```

Comment:
I don't see where this happens in the test. I may just be missing it, but if the test isn't testing that the project owner can delete the method config, then either this comment should be adjusted, or it should test that.

Reply:
`withMethod` fixture redact the method config.